ios/RNPurchases.m

```diff
@@ -204,7 +204,12 @@ - (dispatch_queue_t)methodQueue
     [RCCommonFunctionality invalidatePurchaserInfoCache];
 }
 
-#pragma mark Subscriber Attributes
+RCT_EXPORT_METHOD(setProxyURLString:(nullable NSString *)proxyURLString)
```

Comment:
proxyURLString is still a subscriber attribute, this should be inside the pragma, right?